dataflow/src/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -634,6 +635,18 @@ public boolean containsOfClass(Class<? extends FlowExpressions.Receiver> clazz)
             return false;
         }
 
+        public Receiver getReceiver() {
```

Comment:
Can you add some documentation to these new public methods? Who is allowed to modify the returned objects? Is the returned list modifiable? Should you wrap it in an unmodifiable list?